proto/control-api/src/grpc/api.proto

```diff
@@ -142,6 +144,14 @@ message Member {
   string credentials = 4;
   // Pipeline of this Member.
   map<string, Member.Element> pipeline = 5;
+  // Duration, after which remote RPC client will be considered idle if no
+  // heartbeat messages received.
+  google.protobuf.Duration idle_timeout = 6;
```

Comment:
The terminoligy should be the same as in other places in this file.

We were talking about Member, media server and Client API, and then boom! `RPC client` is here!

It's hard for the reader to understand what is `PRC client` here at all.